File: class12/java/AP1208-Bhaskara.java
public class Bhaskara {
    public static void main(String[] x) {
        double a = 1, b = -5, c = 6;
        double delta = b*b - 4*a*c;
        if (delta >= 0) {
            System.out.println("Raizes: " + (-b + Math.sqrt(delta))/(2*a) + " e " + (-b - Math.sqrt(delta))/(2*a));
        } else System.out.println("Sem raizes reais");
    }
}

File: class12/python/AP1208-Bhaskara.py
import math
a, b, c = 1, -5, 6
delta = b*b - 4*a*c
if delta >= 0:
    x1 = (-b + math.sqrt(delta)) / (2*a)
    x2 = (-b - math.sqrt(delta)) / (2*a)
    print(f"Raizes: {x1} e {x2}")
else:
    print("Sem raizes reais")
